test/common/common/utility_fuzz_test.cc

```diff
@@ -0,0 +1,31 @@
+#include "common/common/utility.h"
+
+#include "test/fuzz/fuzz_runner.h"
+
+#include "absl/strings/string_view.h"
+
+namespace Envoy {
+namespace Fuzz {
+
+DEFINE_FUZZER(const uint8_t* buf, size_t len) {
+
+  uint64_t out;
+  std::string stringBuffer(*buf, len);
```

Comment:
I think you want this to be `(buf, len)`, rather than `(*buf, len)`. Otherwise, it's just creating a fixed string :)

Reply:
Done!